confidential-aci-minecraft/2.MinecraftConfidentialBedrockCMK.json

```diff
@@ -0,0 +1,61 @@
+{
+"$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+"contentVersion": "1.0.0.0",
+"resources": [
+    {
+            "name": "<YOUR DEPLOYMENT NAME>",
+            "type": "Microsoft.ContainerInstance/containerGroups",
+            "apiVersion": "2022-10-01-preview",
+            "location": "[resourceGroup().location]",
+            "properties": {
+                "containers": [
+                    {
+                        "name": "[uniqueString( resourceGroup().id, deployment().name )]",
+                        "properties": {
+                            "image": "itzg/minecraft-bedrock-server",
```

Comment:
what dockerhub repo is this?

Reply:
https://hub.docker.com/r/itzg/minecraft-bedrock-server